gateway-ha/src/main/resources/gateway-ha-persistence-oracle.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE gateway_backend (
+    name VARCHAR(256) PRIMARY KEY,
+    routing_group VARCHAR (256),
+    backend_url VARCHAR (256),
+    external_url VARCHAR (256),
+    active NUMBER(1)
+);
+
+CREATE TABLE query_history (
+    query_id VARCHAR(256) PRIMARY KEY,
+    query_text VARCHAR (256),
+    created NUMBER,
+    backend_url VARCHAR (256),
+    user_name VARCHAR(256),
+    source VARCHAR(256)
+);
+CREATE INDEX query_history_created_idx ON query_history(created);
+
+CREATE TABLE resource_groups (
+    resource_group_id NUMBER GENERATED ALWAYS as IDENTITY(START with 1 INCREMENT by 1),
```

Comment:
I don't know Oracle very well, but would a `LONG` be a better choice than a `NUMBER`? Iiuc a `NUMBER` with unspecified scale supports real numbers as well as integers. Not that it probably matters much

Reply:
a `LONG` in oracle is not actually a numeric data type confusingly enough :) https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Data-Types.html#GUID-F6309DF8-162F-48A4-9454-FEE59EC6644F

I would vote to use `NUMBER` as the data type for the identity column here.

Reply:
After reviewing, I'd agree